fma_benchmark/main.py

```diff
@@ -0,0 +1,240 @@
+import argparse
+import os
+import random
+from typing import List, Dict
+
+from nendo import Nendo, NendoConfig, NendoTrack
+from tqdm import tqdm
+
+# suppress TF warnings
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+PROMPT_TEMPLATE = """Please use the following key and values 
+of a description of an audio file 
+to generate a summary of its contents. 
+Make sure to keep the summary brief, but include all the information 
+especially things like tempo, key, scale and other audio features:"
+"""
+
+QUERY_PROMPT_TEMPLATE = """Please use the following key 
+and values of a description of 
+an audio file to generate a query to search for similar audio files. 
+Here is an example query:
+
+I want a happy song with a fast tempo and a major key.
```

Comment:
"and a major key" is ambiguous... I think "I want a happy song with a fast tempo in key A Major." would be more precise. Most likely it'll work nonetheless because LLMs can do disambiguation but I had to read it twice to understand what's being asked for. Of course, users might "prompt sloppy" and thus the template might be even more "realistic" like it is but I just wanted to point this out